tests/unit/application/services/test_state_updater_storyline_resolution.py

```
@@ -0,0 +1,89 @@
+
+    def setup_method(self):
+        """Set up test fixtures."""
+        self.storyline_repository = Mock(spec=StorylineRepository)
+        self.updater = StateUpdater(
+            bible_repository=Mock(spec=BibleRepository),
+            foreshadowing_repository=Mock(spec=ForeshadowingRepository),
+            storyline_repository=self.storyline_repository,
+        )
+        self.novel_id = NovelId("novel-test")
+
+    def _make_storyline(self, id: str, name: str) -> Storyline:
```

Ruff (0.15.15)

[error] Function argument `id` is shadowing a Python builtin

(A002)